src/localization/bundles/en.json

```diff
@@ -762,5 +765,6 @@
     "table.column.header.server-group" : "Server group",
     "table.column.header.nic-mapping" : "NIC mapping",
     "table.column.header.mac-addr" : "MAC address",
-    "table.column.header.monascaStatus" : "Status"
+    "table.column.header.monascaStatus" : "Status",
+    "table.column.header.serverState": "State"
```

Comment:
@eapodaca Now we have status and state, if both of them are there.. could it be confusing? This is more of a question to think through.